packages/signature-v4a/README.md

```diff
@@ -0,0 +1,13 @@
+# @aws-sdk/signature-v4a
+
+[![NPM version](https://img.shields.io/npm/v/@aws-sdk/signature-v4a/latest.svg)](https://www.npmjs.com/package/@aws-sdk/signature-v4a)
+[![NPM downloads](https://img.shields.io/npm/dm/@aws-sdk/signature-v4a.svg)](https://www.npmjs.com/package/@aws-sdk/signature-v4a)
+
+See: https://github.com/aws/aws-sdk-js-v3/tree/main#functionality-requiring-aws-common-runtime-crt.
+
+## Description
+
+Alias package for `@smithy/signature-v4a`. This package provides a SigV4a-compatible request signer ([`@smithy/signature-v4a`](https://www.npmjs.com/package/@smithy/signature-v4a)) if the request is not regional.
+
+An un-regional request is identified by the `signingRegion` parameter. A region is un-regional if the `signingRegion`
```

Comment:
I dunno if I wrote this myself earlier, but I the terms **regional** **un-regional** should be replaced by **single-region** and **multi-region**.

Sigv4a supposedly can be used when the signing region is a set like `"us-east-1,us-west-2"`. I'm not sure if this is true in practice but it's an aim of it.